### PR TITLE
added api calls for service keys ans team members management

### PR DESCRIPTION
**Pull request by jenjenp**
Added api calls for:
List service keys
Create service key
Get team members list for a service key
Add team members to a service key

### CODE REVIEW

File: services.yaml [hunk 1]
```diff
@@ -45,10 +45,44 @@ paths:
     get:
       tags:
         - General
-      description: "List all services"
+      summary: "List all services"
+      description: "List all services accessible by the user"
+      parameters:
+        - in: query
+          name: service_key_name
```
Comment (chook) [inline services.yaml]: why not simply _name_?

File: services.yaml [hunk 2]
```diff
@@ -45,10 +45,44 @@ paths:
...
+          description: Unauthorized
+          $ref: "#/definitions/UnauthorizedError"
+    post:
+      tags:
+        - General
+      summary: Generate a new service key
+      description: "Create a new service"
+      parameters:
+        - in: body
+          key_name: service
```
Comment (chook) [inline services.yaml]: not sure what is `key_name` and `example` in this space. shouldn't this be a JSON?

File: services.yaml [hunk 3]
```diff
@@ -45,10 +45,44 @@ paths:
...
+          description: service properties
+          type: string
+          example: Pre-prod
+          required: true
+      responses:
+        200:
+          description: New Service Key created
```
Comment (chook) [inline services.yaml]: if the return is a service basic, then this description is not accurate

File: services.yaml [hunk 4]
```diff
@@ -70,29 +104,59 @@ paths:
...
     get:
       tags:
-        - Deployments
-      summary: "List Deployments"
-      description: "List Deployments"
+        - General
+      summary: "List service team members"
+      description: "List service team members"
+      parameters:
+          - in: path
```
Comment (chook) [inline services.yaml]: tab should be 2 spaces

File: services.yaml [hunk 5]
```diff
@@ -70,29 +104,59 @@ paths:
...
+            name: service_id
+            description: Environment to use
+            type: string
+            required: true
+          - in: query
+            name: email
+            description: Team Member Email
```
Comment (chook) [inline services.yaml]: Team member email

File: services.yaml [hunk 6]
```diff
@@ -2056,3 +2137,34 @@ definitions:
       method_desc:
         type: "string"
         example: "([Ljava/lang/String;)V"
+  ServiceBasic:
+    description: Basic information about a service key
+    type: object
+    properties:
+      key_id:
```
Comment (chook) [inline services.yaml]: please remove these key_ prefixes as they don't help :)

**Review by chook — CHANGES_REQUESTED** [review]

File: services.yaml (hunk 1 above)
Comment (jenjenp) [reply services.yaml]: fixed

File: services.yaml (hunk 2 above)
Comment (jenjenp) [reply services.yaml]: fixed

File: services.yaml (hunk 3 above)
Comment (jenjenp) [reply services.yaml]: fixed

File: services.yaml (hunk 4 above)
Comment (jenjenp) [reply services.yaml]: fixed

File: services.yaml (hunk 5 above)
Comment (jenjenp) [reply services.yaml]: fixed

File: services.yaml (hunk 6 above)
Comment (jenjenp) [reply services.yaml]: fixed

**Review by chook — APPROVED** [review]

**Pull request merged**